account_asset_management/models/account_asset_group.py

```diff
@@ -72,7 +73,6 @@ def _name_search(
             ]
             if operator in expression.NEGATIVE_TERM_OPERATORS:
                 domain = ["&", "!"] + domain[1:]
-        rec_ids = self._search(
+        return self._search(
```

Comment:
Has `_name_search` changed its API, returning another different thing now?

Reply:
Maybe odoo example it's a good use case explanation: https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_account.py#L362-L370

Reply:
More than the example, I prefer to see the method itself, its docstring, and if changed, which and why is changed. It's the only source of real truth.

Reply:
Ok, sorry https://github.com/odoo/odoo/blob/14.0/odoo/models.py#L1803-L1816

Reply:
And yes, this is the change of API belonging to 14.0:

https://github.com/odoo/odoo/commit/1f48130d2bd055835ee428a21922331d126285a3

so the change is correct indeed. I'm going to note it in the migration guide.